src/contracts/AbstractARM.sol

```diff
@@ -726,4 +771,132 @@ abstract contract AbstractARM is OwnableOperable, ERC20Upgradeable {
 
         fees = SafeCast.toUint256(assetIncrease) * fee / FEE_SCALE;
     }
+
+    ////////////////////////////////////////////////////
+    ///         Lending Market Functions
+    ////////////////////////////////////////////////////
+
+    /// @notice Owner adds supported lending market to the ARM.
+    /// @param _markets The addresses of the lending markets to add
+    function addMarkets(address[] calldata _markets) external onlyOwner {
```

Comment:
In order to be a safe market for the ARM, it must be:
- up only exchange rate
- no slippage
- no fees.

Don't know if we need to note the properties required anywhere.

Reply:
I've added this comment to the Natspec